Build errors in VS2015

Hi there,

In it's current state, the project does not build in VS2015, the error looks to be related to multiple `AuthorizeNet` packages being specified in packages.config:

```text
Error occurred while restoring NuGet packages: An error occurred while reading file 'd:\misc\sample-code-csharp\packages.config': There are duplicate packages: AuthorizeNet
1>------ Rebuild All started: Project: SampleCode, Configuration: Debug Any CPU ------
1>C:\Program Files (x86)\MSBuild\14.0\bin\Microsoft.Common.CurrentVersion.targets(1819,5): warning MSB3245: Could not resolve this reference. Could not locate the assembly "AuthorizeNet". Check to make sure the assembly exists on disk. If this reference is required by your code, you may get compilation errors.
1>d:\misc\sample-code-csharp\ApplePayTransactions\CreateAnApplePayTransaction.cs(5,7,5,19): error CS0246: The type or namespace name 'AuthorizeNet' could not be found (are you missing a using directive or an assembly reference?)
1>d:\misc\sample-code-csharp\ApplePayTransactions\CreateAnApplePayTransaction.cs(6,7,6,19): error CS0246: The type or namespace name 'AuthorizeNet' could not be found (are you missing a using directive or an assembly reference?)
1>d:\misc\sample-code-csharp\ApplePayTransactions\CreateAnApplePayTransaction.cs(7,7,7,19): error CS0246: The type or namespace name 'AuthorizeNet' could not be found (are you missing a using directive or an assembly reference?)
1>d:\misc\sample-code-csharp\CustomerProfiles\CreateCustomerPaymentProfile.cs(3,7,3,19): error CS0246: The type or namespace name 'AuthorizeNet' could not be found (are you missing a using directive or an assembly reference?)
1>d:\misc\sample-code-csharp\CustomerProfiles\CreateCustomerPaymentProfile.cs(4,7,4,19): error CS0246: The type or namespace name 'AuthorizeNet' could not be found (are you missing a using directive or an assembly reference?)
1>d:\misc\sample-code-csharp\CustomerProfiles\CreateCustomerPaymentProfile.cs(5,7,5,19): error CS0246: The type or namespace name 'AuthorizeNet' could not be found (are you missing a using directive or an assembly reference?)
1>d:\misc\sample-code-csharp\CustomerProfiles\CreateCustomerProfile.cs(3,7,3,19): error CS0246: The type or namespace name 'AuthorizeNet' could not be found (are you missing a using directive or an assembly reference?)
1>d:\misc\sample-code-csharp\CustomerProfiles\CreateCustomerProfile.cs(4,7,4,19): error CS0246: The type or namespace name 'AuthorizeNet' could not be found (are you missing a using directive or an assembly reference?)
1>d:\misc\sample-code-csharp\CustomerProfiles\CreateCustomerProfile.cs(5,7,5,19): error CS0246: The type or namespace name 'AuthorizeNet' could not be found (are you missing a using directive or an assembly reference?)
...
========== Build: 0 succeeded, 1 failed, 0 up-to-date, 0 skipped ==========
```

I suspect the newer version of NuGet that VS2015 ships with does not handle duplicate references, but previous versions did.

I'm happy to submit a PR removing the additional reference.

Thanks!

--- a/packages.config
+++ b/packages.config
@@ -1,5 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="AuthorizeNet" version="1.8.8" targetFramework="net45" />
-  <package id="AuthorizeNet" version="1.8.8" targetFramework="net40" />
+  <package id="AuthorizeNet" version="1.8.8" />
 </packages>

